plugins/auth-backend-module-pinniped-provider/src/authenticator.ts

```diff
@@ -54,21 +57,78 @@ const rfc8693TokenExchange = async ({
 };
 
 /** @public */
-export const pinnipedAuthenticator = createOAuthAuthenticator({
-  defaultProfileTransform: async (_r, _c) => ({ profile: {} }),
-  async initialize({ callbackUrl, config }) {
+export const CACHED_PROMISE_EXPIRATION = 3600;
```

Comment:
I don't think this needs to be exported, but I like having it as a constant like this. I'd also like a better name though, something like `OIDC_METADATA_TTL_SECONDS`